cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie_list"),
```

Comment:
I would rename name="movie_list" to name="movie-list" (using hyphen). The same for the next line.